### PR TITLE
[voq/systemlag] LAG ID allocator defintions

### PR DESCRIPTION
**Pull request by vganesan-nokia**
Signed-off-by: vedganes <vedavinayagam.ganesan@nokia.com>

Changes include: Definitions for LAG id allocator and change in
runRedisScript api to process integer returned by script run in the
redis.

### CODE REVIEW

**Comment by anshuv-mfst** [comment]
@judyjoseph - could you please review the PR, thanks.

File: common/schema.h [hunk 1]
```diff
@@ -315,6 +315,11 @@ namespace swss {
 #define CHASSIS_APP_SYSTEM_INTERFACE_TABLE_NAME     "SYSTEM_INTERFACE"
 #define CHASSIS_APP_SYSTEM_NEIGH_TABLE_NAME         "SYSTEM_NEIGH"
 
+#define CHASSIS_APP_LAG_ID_START_NAME               "SYSTEM_LAG_ID_START"
```
Comment (judyjoseph) [inline common/schema.h:318]: These definitions can be removed as I see them not used anywhere. We just have the TABLE_NAMES mostly in this schema file.

Comment (vganesan-nokia) [reply common/schema.h:318]: Yes. I'll remove them.

Comment (vganesan-nokia) [reply common/schema.h]: Fixed.

**Review by lguohan — APPROVED** [review]

**Pull request merged**